src/app/dataset-flow/dataset-flow-details/tabs/flow-details-history-tab/flow-details-history-tab.component.html

```diff
@@ -12,11 +12,18 @@
                     <div class="col-md-6 d-flex position-relative">
                         <div class="history-item">
                             <img
-                                *ngIf="lastIndex && flowDetails.status !== FlowStatus.Finished"
+                                *ngIf="lastIndex && flowDetails.status === FlowStatus.Waiting"
                                 src="assets/images/hourglass.gif"
                                 alt="hourglass"
                                 style="width: 15px"
-                                class="hourglass-img text-muted"
+                                class="hourglass-img"
+                            />
+                            <img
+                                *ngIf="lastIndex && flowDetails.status === FlowStatus.Running"
+                                src="assets/images/gear.gif"
+                                alt="gear"
+                                style="width: 15px"
+                                class="gear-img"
```

Comment:
Move the logic to component. Return an optional struct with the attributes and use it in template for single `<img>` tag